packages/generator/src/parser/service-parser.ts

```diff
@@ -189,7 +189,7 @@ function transformServiceMetadata(
 function apiBusinessHubMetadata(
   swagger?: SwaggerMetadata
 ): ApiBusinessHubMetadata | undefined {
-  if (!swagger) {
+  if (!swagger || !swagger?.basePath) {
```

Comment:
So, this is not related to the open api thing, but rather to avoid the error message in line 213?

Reply:
yes